### ISSUE TITLE
Terminal versions after 1.14.2282.0 won't start

### ISSUE THREAD

**Issue by KyleKorndoerfer** [microsoft/terminal#14212]
### Windows Terminal version

1.14.2282.0

### Windows build number

10.0.22000.0

### Steps to reproduce

I can only run WT at version 1.14.2282.0. If I install any newer version it fails to start (no window appears, no process is listed in Task Manager, etc.). Even when attempting to start from the command prompt/PowerShell with wt.exe, nothing is reported back to the console. This would be fine except it appears that Windows Terminal is getting updated in the background somehow so each time I restart my Azure DevTestLabs VM I need to uninstall, reinstall, & overwrite settings.

Microsoft Store is blocked on my VM so I have to manually install WT from the GitHub releases and would not assume that it would be auto-updated.

### Expected Behavior

launch as normal and not be updated automatically

### Actual Behavior

Any version above 1.14.2282.0 fails to start and results in an Application Error in the Windows Application Event Log as follows:

```xml
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" /> 
    <EventID Qualifiers="0">1000</EventID> 
    <Version>0</Version> 
    <Level>2</Level> 
    <Task>100</Task> 
    <Opcode>0</Opcode> 
    <Keywords>0x80000000000000</Keywords> 
    <TimeCreated SystemTime="2022-10-13T14:40:52.2829793Z" /> 
    <EventRecordID>10596</EventRecordID> 
    <Correlation /> 
    <Execution ProcessID="13228" ThreadID="0" /> 
    <Channel>Application</Channel> 
    <Computer>(redacted)</Computer> 
    <Security /> 
  </System>
  <EventData>
    <Data>WindowsTerminal.exe</Data> 
    <Data>1.15.2209.28003</Data> 
    <Data>6334cd9b</Data> 
    <Data>Windows.UI.Xaml.dll</Data> 
    <Data>10.0.22000.978</Data> 
    <Data>82655b66</Data> 
    <Data>c000027b</Data> 
    <Data>00000000004a3e02</Data> 
    <Data>31f8</Data> 
    <Data>01d8df11cae1f6ed</Data> 
    <Data>C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.15.2713.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe</Data> 
    <Data>C:\Windows\System32\Windows.UI.Xaml.dll</Data> 
    <Data>17440c44-24fe-4338-add1-cb36a6c31dee</Data> 
    <Data>Microsoft.WindowsTerminal_1.15.2713.0_x64__8wekyb3d8bbwe</Data> 
    <Data>App</Data> 
  </EventData>
</Event>
```

**Comment by DHowett** [comment]
Oof! Is there a chance that you're running "Windows N" (the no-media SKU) on that host?

**Comment by KyleKorndoerfer** [comment]
Yes, **System > About** is reporting Windows 11 Enterprise N (21H2)

**Comment by DHowett** [comment]
Argh! Thank you for reporting it and confirming.

This is a /duplicate of #14104, for which a fix is being released shortly.

If you need to get unblocked _today_, installing the media feature pack will help. I'm sorry about the issue.
